### DISCUSSION TITLE
Users' Path prompts in Terminal window

### DISCUSSION THREAD

**Discussion by rweilert** [ltsp/ltsp#334] [Q&A]
This is under Ubuntu Mate 18.04 with LTSP5:

Recently I saw that on all other users' terminal windows except mine, there is merely a "$" as a prompt, but no path. On my account, everything is shown as expected. I searched a bit within the profile files everywhere ($HOMES and /etc), but I found no difference between mine and those of the other users. (This is within the Terminal window of Mate.) And I cannot remember if I ever changed anything on my side.

What might be the reason, is there perhaps some configuration file I overlooked?

Thank you for any hint.

Regards
Rolf

**Comment by alkisg** [comment]
Run `cat /etc/passwd` and check the user's shells. Is it /bin/bash or /bin/sh?
If it's /bin/sh, then you didn't use an appropriate method when adding the users...

**Comment by rweilert** [comment]
Oh, you are right... :( all except me have /bin/sh

Aaaaargh! Should have known earlier. That's my script for adding users.

Anyway... Thanks for your help!

Rolf
